README.md

```diff
@@ -65,6 +65,14 @@ Rich text elements in Kentico Cloud can contain links to other content items. It
 
 All content retrieved from Kentico Cloud is by default [cached](https://github.com/Kentico/cloud-boilerplate-net/blob/master/src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs) for five minutes. You can change the time by overriding the value of `CacheTimeoutSeconds` (e.g. in appsettings.json).
 
+The [CachedDeliveryClient](https://github.com/Kentico/cloud-boilerplate-net/blob/master/src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs) class is a simple wrapper around the original [https://github.com/Kentico/delivery-sdk-net/blob/master/KenticoCloud.Delivery/DeliveryClient.cs](DeliveryClient). It uses an [https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.extensions.caching.memory.imemorycache](IMemoryCache) caching, currently implemented with the [https://docs.microsoft.com/en-us/dotnet/api/system.runtime.caching.memorycache?view=netframework-4.7](MemoryCache) object.
+
+The `CachedDeliveryClient` has the same methods as the original `DeliveryClient`. Upon invoking these methods, it stores the results into the in-memory cache, for the aforementioned amount of time.
+
+As an in-memory caching mechanism, it is perfectly fine for non-load-balanced apps, or for apps configured with [https://forums.asp.net/t/1892952.aspx?What+is+sticky+session+](sticky sessions). For other scenarios, a distributed cache should be used instead.
+
+**Note**: Speed of the Delivery/Preview API service is already tuned up because the service uses a geo-distributed CDN network for most of the types of requests. Therefore, the main advantage of caching in Kentico Cloud is not speed but lowering the amount of requests needed (See [pricing](https://kenticocloud.com/pricing) for details).
```

Comment:
"caching in Kentico Cloud" -> caching in Kentico Cloud <applications|boilerplate|something..>